### PR TITLE
Feature/i18n update

### PR DESCRIPTION
**Pull request by Pe5h4**
Changes made as a result of review on branch https://github.com/TeskaLabs/asab-webui/tree/feature/i18n . It concerns the demo application and the result is facilitation for user when adding translations to its project. The `locales` folder was added into the demo application as an example to avoid misleading user when implementing translations.

### CODE REVIEW

File: demo/public/locales/en-US/translation.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"Welcome to React": "Vitejte v Reactu",
```
Comment (ateska) [inline demo/public/locales/en-US/translation.json:2]: This is not english (and I'm not sure if we need `en-US` folder

File: demo/public/locales/en/translation.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+	"Welcome to React": "Vitejte v Reactu",
```
Comment (ateska) [inline demo/public/locales/en/translation.json:2]: Must be english.

**Review by ateska — CHANGES_REQUESTED** [review]
Small changes in the text, then mergeable.

File: demo/public/locales/en-US/translation.json (hunk 1 above)
Comment (Pe5h4) [reply demo/public/locales/en-US/translation.json:2]: Concerning `en-US` folder - it does not need to be there, but if it is not added, then we have a warning with failed localizations ```something went wrong loading 
Array [ "failed parsing /locales/en-US/translation.json to json" ]```

**Review by ateska — APPROVED** [review]

**Pull request merged**